### DISCUSSION TITLE
How to create breakpoint for height? e.g @media (max-height: 922px)

### DISCUSSION THREAD

**Discussion by nicitaacom** [tailwindlabs/tailwindcss#12788] [Help] [answered]
I know that I can create `min-[1800px]:top-[34%]
hot to create for min height because example above work for width

I want if <953px height top-[34%]

Is it possible with tailwind?

**Comment by wongjn** [comment, marked as answer]
Yep! You could use an [arbitrary variant](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-variants), `[@media(max-height:953px)]:top-[34%]` or you could have it as [a `screen` value in your Tailwind configuration](https://tailwindcss.com/docs/screens#custom-media-queries):
```js
screens: {
  foo: {
    raw: '(max-height: 953px)'
  }
}
```
```html
<div class="foo:top-[34%]">
```
See both examples in [this Tailwind Play](https://play.tailwindcss.com/2fFvD0RlH1) (open the *Generated CSS* panel).

**Reply by nicitaacom** [reply]
Thank you!

**Reply by nicitaacom** [reply]
Ask chatGPT
And provide more information that might help us to help you

**Comment by nicitaacom** [comment]
Also I can create something like this
```html
[@media_((min-height:940px)_and_(min-width:1800px))]:top-[34%]
```
![image](https://github.com/tailwindlabs/tailwindcss/assets/39565703/14ed3e06-4420-4013-8f3a-02cb115da20f)
